### DISCUSSION TITLE
First call of the Working Group: 15th Dec 2021

### DISCUSSION THREAD

**Discussion by ruphy** [publiccodeyml/publiccode.yml#110] [Announcements]
Hi all,

it's my pleasure to convene the first meeting of the publiccode.yml Working Group, which will take place on Wednesday 15th December 2021 at 15.00 GMT (convert it to your own timezone here: https://www.inyourowntime.zone/2021-12-15_16.00_Europe.Berlin).

Over the long run I would like to use solely free software solutions to hold our proceedings (suggestions welcome!), but I propose, for the sake of simplicity, to meet over Google Meet for this very first call, which also allows phone-in guests.

The link to join the call: https://meet.google.com/khh-zhay-wuq
If you want to drop in by phone: https://tel.meet/khh-zhay-wuq?pin=5201083101348

I'm happy to send direct invites via mail to whoever needs one, or to publish the event in some format that can be useful. I am also happy to gather feedback on how the call itself was organized: feel free to share thoughts on how the process can be improved for the next times.

The goal of this call is going to be to briefly get to know each other and to kickstart the open governance model, with potentially some decision on the organization model we'd like to adopt. As @bfabio and I are working to make proposals in this direction and at the same time create a slightly more structured agenda, I invite you to raise any topic which you feel important to adress in our first gathering by posting here.

I hope to see you all soon!
Riccardo

**Comment by devinbalkind** [comment]
Here are some governance options/templates made by some thoughtful academics who spend a lot of time thinking about cooperation online, distributed organizing, etc. Might be of some use: https://communityrule.info/templates/

**Reply by ruphy** [reply]
this is a wonderful resource, thank you!!!

**Comment by fnogatz** [comment]
Out of curiosity, is there some kind of notes somewhere of yesterday's first call of the WG? :innocent:

**Reply by bfabio** [reply]
Pinging @ruphy for the minutes of the call :)

**Reply by ruphy** [reply]
it's coming today @bfabio :-) I just needed a revision

**Comment by ruphy** [comment]
## Live participants to the call:

Fabio Bonelli (maintainer), Riccardo Iaconelli (chair), Jan Ainali, Boris Van Hoytema, Bastien Guerry, Johan Linåker, Jonas Södergren, Leonardo Favario, Alessandro Sebastiani

## Minutes:

- Introduction by Riccardo on the open nature of the group and the desire to participate. All participants were invited to briefly introduce themselves.
- Fabio presented the new open governance structure: the key principle is to achieve unanimity with some exceptions for national sections and to avoid deadlocks. This document is in the repository (in the governance/ folder) and everyone is invited to read through it and propose improvements.
- All the activities will happen on Github as asynchronously as possible, we'll host regular meetings as a ritual to formally acknowledge decisions taken on the pull requests. Every decision will be based on Pull Requests after an optional discussion on this forum.
- We switched our main domain to a new one: publiccode.tools / yml.publiccode.tools
- It is a stable url that always points to the latest version, we are still not sure how to version it

## Next steps

- We strive for another 0.x release in 2-3 months, possibly backward compatible, and another breaking 0.x a while later. We aim for releasing 1.0 at some time in 2022 as our first major release under open governance.
- Bastien Guerry gave everyone a quick introduction on the plans of the French government to adopt the format. They should have a resource dedicated to catalogue/PublicCode.yml development starting roughly in the next month.
- The steering committee is officially formed (see [the specific page](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/charter.md#steering-committee-publiccodeymlsteering-committee)). This is only a committee for the final decisions, but the development will happen in agreement with the broader community
- We're looking into repository automation to try and make the voting process automatic
- The Italian team is working on making the tools around the standard more and more international so that they can be reused widely. Help from the community is appreciated!

## Open discussion, questions and answers

- A point was raised on the scope of PublicCode, whether it is meant only for software created by the public or for every software meant for public usage. The scope was defined as all public software, delegating to the individual frontends built on the standard the responsibility of filtering as needed software built by public agencies
- A point was raised on the possible clash of naming with the products of the Foundation of Public Code. We acknowledge our mutual support and the willingness to resolve these issues as they emerge

**Reply by ruphy** [reply]
cc @fnogatz ;-)

**Reply by libremente** [reply]
Thanks @ruphy for the minutes! :+1:

**Comment by Ainali** [comment]
Do we have a date for the next working group? We would like to discuss if and how the compliance to Standard for Public Code could be expressed inside of publiccode.yml.

**Reply by bfabio** [reply]
@Ainali there's no date yet, but we can start the discussion here if you'd like to

**Reply by Ainali** [reply]
Started here. https://github.com/publiccodeyml/publiccode.yml/discussions/136

**Comment by ruphy** [comment]
@Ainali I would like to support what @bfabio said, let's try to keep discussion asyncronous if we can. As we're spread on many different timezones it would make the process much easier. Ideally, I'd like to get to the point where calls are obsoleted by the process that runs here

**Reply by Ainali** [reply]
Okay! Does that mean that the first call was also the last?

**Reply by bfabio** [reply]
@Ainali it's much like an experiment to try and keep the whole process lean and informal (and also open to contributors not participating to the calls) and see how it goes.

But if you feel there's a need for a call on a specific topic or to talk about something on certain occasions, let's arrange that!
